src/App.vue

```diff
@@ -105,7 +105,7 @@
                 :offset="{ top: 86, bottom: 12 }"
               >
                 <!-- Corrections still use the basic Fee Summary component -->
-                <aside v-if="isCorrectionFiling && getFilingData.filingTypeCode && getFilingData.entityType">
+                <aside v-if="isCorrectionFiling && correctionHasFilingData ">
```

Comment:
There is a space at the end and before the double-quote.

Reply:
fix confirmed